src/Elliptic/Systems/SelfForce/Scalar/Equations.hpp

```diff
@@ -0,0 +1,112 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <pup.h>
+
+#include "DataStructures/ComplexDataVector.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "DataStructures/Variables.hpp"
+#include "DataStructures/VariablesTag.hpp"
+#include "Elliptic/Systems/SelfForce/Scalar/Tags.hpp"
+#include "NumericalAlgorithms/DiscontinuousGalerkin/Tags.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/MakeWithValue.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace ScalarSelfForce {
+
+/*!
+ * \brief The first-order flux
+ * $F^i=\{\partial_{r_\star}, \alpha \partial_{\cos\theta}\}\Psi_m$.
+ */
+void fluxes(gsl::not_null<tnsr::I<ComplexDataVector, 2>*> flux,
+            const Scalar<ComplexDataVector>& alpha,
+            const tnsr::i<ComplexDataVector, 2>& field_gradient);
+
+/*!
+ * \brief The first-order flux on an element face
+ * $F^i=\{n_{r_\star}, \alpha n_{\cos\theta}\}\Psi_m$.
+ */
+void fluxes_on_face(gsl::not_null<tnsr::I<ComplexDataVector, 2>*> flux,
+                    const Scalar<ComplexDataVector>& alpha,
+                    const tnsr::I<DataVector, 2>& face_normal_vector,
+                    const Scalar<ComplexDataVector>& field);
+
+/*!
+ * \brief The source term $\beta \Psi_m + \gamma_i F^i$.
+ */
+void add_sources(gsl::not_null<Scalar<ComplexDataVector>*> source,
+                 const Scalar<ComplexDataVector>& beta,
+                 const tnsr::i<ComplexDataVector, 2>& gamma,
+                 const Scalar<ComplexDataVector>& field,
+                 const tnsr::I<ComplexDataVector, 2>& flux);
+
+struct Fluxes {
```

Comment:
Document these structs so they at least show up in the dox? Maybe just `\copydocs` from the functions?